mack/__init__.py

```diff
@@ -160,7 +164,8 @@ def drop_duplicates_pkey(
     for required_column in required_columns:
         if required_column not in data_frame_columns:
             raise TypeError(
-                f"The base table has these columns '{data_frame_columns}', but these columns are required '{required_columns}'"
+                f"The base table has these columns '{data_frame_columns}', but these columns "
```

Comment:
I don't like this change so much.  Can we go for longer lines?

Reply:
I removed this black line formatting now. @MrPowers  it's no longer in this broken lines.